docs/sources/setup-grafana/configure-security/configure-authentication/google/index.md

```diff
@@ -98,3 +99,26 @@ We do not currently sync roles from Google and instead set the AutoAssigned role
 # ..
 skip_org_role_sync = true
 ```
+
+### Configure team sync for Google OAuth
+
+> Available in Grafana v10.1.0 and later versions.
```

Comment:
Because we version our docs, we don't need to add this kind of note. You can remove it.

Reply:
Hey @chri2547 I'm ok with removing but we end up getting PRs to re-add as users are used to using the latest version of the documentation. Could you provide further guidance?

Example: https://github.com/grafana/grafana/pull/69419

Reply:
Hi @Jguer , I checked with the team because you made a good point that I hadn't considered. Our guidance is to keep these kinds of notes in, and when they get stale (for example, 9.5 docs refer to 7.x), remove them as we come across them.